src/stdlib/.gitignore

```diff
@@ -0,0 +1,2 @@
+README.md
+CHANGELOG.md
```

Comment:
I just gitignored these instead of worrying about removing them from the working tree inside the action